### DISCUSSION TITLE
xDrip+ readings from 2 different sensors? 2 Diabetics' readings on 1 screen

### DISCUSSION THREAD

**Discussion by kssxs** [NightscoutFoundation/xDrip#2307] [General]
Hey all! My son and I are both Type 1 Diabetics, and have recently started using xDrip+. I have it installed on my Samsung phone and Watch 5 Pro (using Wearable Widgets) and he has it on his Galaxy S6 tablet and it's working for both of us just fine on our individual devices.

My question is this: Can I have both my blood sugar and his tracked on my devices so that I can see both of our BG's at the same time? I would love to be able to use 2 widgets on my screen (1 for each of us). How can that be setup, if it's even possible?  I saw that there are variant versions if I needed to have to instances installed, but is that the right way of doing this?

Thanks in advance!

**Comment by Navid200** [comment]
Yes, please try a variant on your phone to follow your son.
And tell us if you have any issues.
https://navid200.github.io/xDrip/docs/Variants.html

**Comment by kssxs** [comment]
Done!  This is working perfectly, thank you!
